source/SIL.AppBuilder.Portal.Frontend/src/lib/collection.ts

```diff
@@ -16,3 +16,14 @@ export const applyNumberOfTimes = (x, f) => {
 
   return result;
 };
+
+export function compareVia(accessor: any) {
```

Comment:
what is compareVia? is that a good name?

Reply:
did you see the usage? 
it's a comparator function :)

Reply:
Yeah I do know what it does, just saying that name sounds vague, perhaps comparator?